scripts/case_selection/shared_fxns.R

```diff
@@ -289,11 +289,13 @@ remap_patient_characteristics <- function(clinical, existing_patients, ethnicity
   return(patient_df)
 }
 
-#' Check for missing values
+#' Check for missing values in naaccr columns
 #'
 #' @param data The data frame to check against
 #' @param columns The target columns
 check_for_missing_values <- function(data, columns) {
+  # filter out CHOP, PROV, JHU centers with known NAs
+  data <- data[!grepl("CHOP|PROV|JHU", data$genie_patient_id), ]
```

Comment:
It's ok to leave it in.  In the future, even after we fix it on main GENIE, there may be issues

Reply:
Once it's fixed on main GENIE, we can drop this row.

Reply:
Can we add to the AC of that ticket that rixing created? To drop this row

Reply:
Sure. On it.